index.html

```diff
@@ -259,8 +259,8 @@ <h4 class="panel-title">
           <a href="#" ng-click="toggleSidebar()">
             <img alt="brand" src="assets/img/favicon.png" title="Connected to {{ settings.host }}:{{ settings.port}}">
           </a>
-          <span class="badge" ng-show="unread > 0">{{unread}}</span>
-          <span class="badge danger" ng-show="notifications > 0">{{notifications}}</span>
+          <span class="badge" ng-class="{'badge-show': showUnreadBadge}" ng-show="unread > 0">{{unread}}</span>
+          <span class="badge danger" ng-class="{'badge-show': showUnreadBadge}" ng-show="notifications > 0">{{notifications}}</span>
```

Comment:
you can use `settings.alwaysunreadbadge` directly here and save yourself the callback further down